.mailmap

```diff
@@ -487,6 +487,7 @@ Demian Wassermann <demian@bwh.harvard.edu>
 Denis Ivanenko <ivanenko@ucu.edu.ua> Jack <ivanenko@ucu.edu.ua>
 Denis Ivanenko <ivanenko@ucu.edu.ua> LilJohny <ivanenko@ucu.edu.ua>
 Denis Rykov <rykovd@gmail.com>
+Denis Shuvalov <shuvalovds@gmail.com>
```

Comment:
you need to run the `.mailmap` file by the following command
`python bin/mailmap_check.py`